.travis.yml

```diff
@@ -1,161 +1,80 @@
+sudo: false
```

Comment:
the travis config tab on the build page warns that sudo is deprecated and no longer needed